src/main/java/seedu/address/model/person/Gender.java

```diff
@@ -8,7 +8,7 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidGender(String)}
  */
 public class Gender {
-    public static final String MESSAGE_CONSTRAINTS = "Gender should be 'M' or 'F'";
+    public static final String MESSAGE_CONSTRAINTS = "Gender should be 'M/m' or 'F/f'";
```

Comment:
Maybe clearer to indicate the part in quotes as literal input:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Gender should be 'M'/'m' or 'F'/'f'";
```